config/environments/development.rb

```diff
@@ -7,6 +7,13 @@
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
+  config.assets.css_compressor = nil
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.perform_deliveries = true
+
+  user_confirmation_url(@user, only_path: true)
```

Comment:
- [ ] [OPTIONAL] Hello team, let's take out those particular settings as it stops users from running the application. I had to comment it out before I could access the project.

user_confirmation_url(@user, only_path: true)

Reply:
Got it Thank you